web/docusaurus.config.js

```diff
@@ -82,34 +82,14 @@ const config = {
             type: "html",
             position: "right",
             value:
-              '<a href="/docs/installation/install/manifest" style="background-color: #FF8803; color: #FFF; height: 30px; width: 150px; text-decoration: none; display: inline-flex; align-items: center; border-radius: 30px 30px 30px 30px"' +
+              '<a href="/docs/installation/install/manifest" style="background-color: #FF8803; color: #FFF; height: 30px; width: 150px; text-decoration: none; display: flex; align-content: center; justify-content: center; border-radius: 30px 30px 30px 30px"' +
               "onmouseover=\"this.style.backgroundColor='#FFA229';\"" +
               "onmouseout=\"this.style.backgroundColor='#FF8803';\">" +
-              '<h3 style="margin: 0px auto;">' +
+              "<h3>" +
               "Get started" +
               "</h3>" +
               "</a>",
           },
-          {
-            href: "https://github.com/cyclops-ui/cyclops",
-            position: "right",
-            className: "header-github-link",
-          },
-          {
-            href: "https://www.linkedin.com/company/cyclops-ui",
-            position: "right",
-            className: "header-linkedin-link",
-          },
-          {
-            href: "https://x.com/CyclopsUI",
-            position: "right",
-            className: "header-x-link",
-          },
-          {
-            href: "https://discord.com/invite/8ErnK3qDb3",
-            className: "header-discord-link",
-            position: "right",
-          },
```

Comment:
We need the links to our socials, so we can't remove them. You can add the right padding to `header-discord-link` class to split the theme switch icon from the socials

Reply:
can we change the icons style (svg)? @petar-cvit .

Reply:
How would you change them?

Reply:
I can change by adding different svgs by setting type to html

Reply:
It look like this now:
![image](https://github.com/user-attachments/assets/1e599638-b723-4155-9f45-2a3f210de704)